backend/src/test/resources/application.properties

```diff
@@ -9,6 +9,8 @@ puris.response.serverendpoint=${PURIS_RESPONSE_SERVERENDPOINT:http://customer-ba
 puris.response.apiassetid=${PURIS_RESPONSE_APIASSETID:response-api-asset}
 puris.statusrequest.apiassetid=${PURIS_STATUSREQUEST_APIASSETID:statusrequest-api-asset}
 puris.statusrequest.serverendpoint=${PURIS_STATUSREQUEST_SERVERENDPOINT:http://customer-backend:8081/catena/item-stock/status}
+puris.useframeworkagreement=${PURIS_USEFRAMEWORKAGREEMENT:false}
```

Comment:
Please update to puris.frameworkagreement style

Reply:
Done